src/pipeline.ml

```diff
@@ -367,6 +367,8 @@ let tarides ?app ?notify:channel ?filter ~sched ~staging_auth () =
         ~archs:[`Linux_x86_64; `Linux_arm64] ~options:include_git;
       docker "Dockerfile.worker" ["live-worker",    "ocurrent/ocluster-worker:live", []]
         ~archs:[`Linux_x86_64; `Linux_arm64; `Linux_ppc64; `Linux_s390x; `Linux_riscv64] ~options:include_git;
+      docker "Dockerfile"        ["live-monitor",   "ocurrent/clarke:live", []] 
+        ~archs:[ `Linux_x86_64 ] ~options:include_git;
```

Comment:
May as well build for all the same architectures as `ocluster-worker`.

Reply:
Forced pushed and done 👍